### PR TITLE
AC Automated Test - LRAC-11529 | master

### PR DESCRIPTION
**Pull request by manuferreira**
https://issues.liferay.com/browse/LRAC-11529 (automation ticket)
https://issues.liferay.com/browse/LRAC-7916 (bug ticket)

Hello, in case you guys are wondering why this case has another automation ticket ([LRAC-9113](https://issues.liferay.com/browse/LRAC-9113)) the old one was closed because at that time it wasn't possible to create a new workspace via automation.

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 1]
```diff
@@ -145,13 +145,54 @@ definition {
 		}
 	}
 
-	@description = "Feature ID: Legacy | Automation ID: LRAC-9113 | Test Summary: It is possible to create multiple properties in different workspaces"
-	@ignore = "true"
+	@description = "Bug: 7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"
 	@priority = "4"
 	test CanCreateMultiplePropertiesInDifferentWorkspaces {
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:150]: Thanks for working on this! Can we rename this test to something like `CanCreateSamePropertyNameInDifferentWorkspaces`? I think it describes the test better. What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 2]
```diff
@@ -145,13 +145,54 @@ definition {
 		}
 	}
 
-	@description = "Feature ID: Legacy | Automation ID: LRAC-9113 | Test Summary: It is possible to create multiple properties in different workspaces"
-	@ignore = "true"
+	@description = "Bug: 7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:148]: Hey @manuferreira, you forgot LRAC in the bug ticket:

`@description = "Bug: LRAC-7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 3]
```diff
@@ -145,13 +145,54 @@ definition {
...
 	@priority = "4"
 	test CanCreateMultiplePropertiesInDifferentWorkspaces {
+		var acProperty = "AC Property Name";
+		var acWorkspace = "AC New Workspace";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:152]: Hey @manuferreira, could you change the name of the workspace to **New AC Workspace**?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 4]
```diff
@@ -251,6 +242,53 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"
+	@priority = "4"
+	test CanCreateSamePropertyNameInDifferentWorkspaces {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:247]: Hey @manuferreira, could you put this test to run in isolation? Tests that create a new workspace need to be run in isolation so as not to interfere with other tests. After this [ticket](https://issues.liferay.com/browse/LRAC-11544?jql=project%20%3D%20LRAC%20AND%20fixVersion%20%3D%20release-3.3.3) we will be able to delete workspace and these cases will not need to be executed in isolation

**Comment by manuferreira** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
LGTM Manu. Thanks for it 🎉

**Review by joshitagaki — APPROVED** [review]
Thanks @manuferreira! LGTM!

**Comment by calvinctvo** [comment]
Hey guys! Thanks for reviewing! I'm forwarding this PR

**Comment by calvinctvo** [comment]
ci:forward

**Pull request closed without merging**